remote-api/src/main/java/org/alfresco/rest/api/impl/NodesImpl.java

```diff
@@ -3420,7 +3420,7 @@ public Node unlock(String nodeId, Parameters parameters)
     @Override
     public DirectAccessUrl requestContentDirectUrl(NodeRef nodeRef, boolean attachment, Long validFor)
     {
-        DirectAccessUrl directAccessUrl = contentService.requestContentDirectUrl(nodeRef, attachment, validFor);
+        DirectAccessUrl directAccessUrl = contentService.requestContentDirectUrl(nodeRef, ContentModel.PROP_CONTENT, attachment, validFor);
```

Comment:
This looks good. What about all other usages of the deprecated methods ?

Reply:
Do you mean I should find methods in other projects such as rest-api-explorer? Because according to IntelliJ, there are no more usages in this project.
![image](https://user-images.githubusercontent.com/42536841/144083521-b5218dbe-7ae3-42a4-9d75-05e93a651f58.png)

Reply:
After quick review with @aymanthefirst it looks like all REST APIs go via the NodesImpl after all (including for Deleted Nodes &/or Renditions).